Gateway/Request/APMBuilder.php

```diff
@@ -16,6 +17,7 @@
 use Omise\Payment\Model\Config\Truemoney;
 
 use Omise\Payment\Observer\ConveniencestoreDataAssignObserver;
+use Omise\Payment\Observer\FPXDataAssignObserver;
```

Comment:
Very small, but the capitalization approach for FPX here (`FPX`) is different than the one above (`Fpx`). `FpxDataAssignObserver` would be more consistent.